src/main/resources/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder/config.jelly

```diff
@@ -16,10 +16,14 @@
     <f:entry field="manifestPattern" title="${%Kubernetes Manifests}">
         <f:textbox/>
     </f:entry>
-    <!-- TODO(craigatgoogle): re-enable once these QOL features are complete.
     <f:entry field="verifyDeployments" title="${%Verify Deployments}">
         <f:checkbox name="verifyDeployments" checked="${it.verifyDeployments}"/>
     </f:entry>
+
+    <f:entry field="verifyTimeoutInMinutes" title="${%Verify Timeout in Minutes}">
```

Comment:
This should be made into an optional block (see [this example](https://wiki.jenkins.io/display/JENKINS/Jelly+form+controls)). When running locally I saw verification errors for this textbox despite not really needing it because the checkbox was not checked. It would be smoother if this was hidden unless the user checks "Verify Deployments".